challenges/2022-10-11-blast-from-the-past/solutions/javascript/brehberg/brehberg.js

```diff
@@ -0,0 +1,33 @@
+// # Find Three Largest Numbers
+//
+// Write a function that takes in an array of at least three integers and, without
+// sorting the input array, returns a sorted array of the three largest integers in
+// the input array. The function should return duplicate integers if necessary.
+//
+// ## Business Rules/Errata
+//
+// - The input array should have at least three integers. If it does not, you should
+//   return a null value.
+// - You may not sort the input array
+// - The function should handle duplicate integers; for example `[10, 5, 9, 10, 12]`
+//   should return `[10, 10, 12]`
+// - Constant space -> you will return a new array of 3 integers, and this will be the
+//   only new data structure you create.
+// - Linear time -> you should solve this problem by only passing through the array a
+//   single time.
+
+function findThreeLargestNumbers(arr) {
+  if (arr.length < 2) return null;
```

Comment:
I think this should be arr.length < 3 right? length are usually 1 based

Reply:
Means we have poor test cases :D

Reply:
Correct on both counts :) 
My solution has been updated and a few failing JS tests were added